src/components/signed-out/signed-out-controller.ts

```diff
@@ -0,0 +1,5 @@
+import { Request, Response } from "express";
+
+export function signedOutGet(req: Request, res: Response): void {
+  res.render("signed-out/index.njk");
```

Comment:
i would clear all cookies on the res if there are any

Reply:
Which cookies? Guess we should leave cookie preferences?

Reply:
deleting all cookies except 'cookies_preferences_set' and 'lng' as these are user preference rather than session related.